src/framework/uicomponents/view/topleveldialog.cpp

```diff
@@ -0,0 +1,67 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include "topleveldialog.h"
+
+#include <QApplication>
+
+using namespace mu::uicomponents;
+
+TopLevelDialog::TopLevelDialog(QWidget* parent)
+    : QDialog(parent)
+{
+    setAttribute(Qt::WA_NativeWindow, true);
```

Comment:
I recall this window attribute was specific to the VST editors, because their content needs to be attached to a native window. I don't expect it will be necessary for all top level dialogs.

Reply:
Thank you! Fixed. But unfortunately, I had to remove the setTransientParent call from the constructor (now it will be called from TopLevelDialog::event). In the constructor, this call causes the crash of the application if the WA_NativeWindow attribute is not set.